Sources/RealModule/ApproximateEquality.swift

```diff
@@ -0,0 +1,154 @@
+//===--- ApproximateEquality.swift ----------------------------*- swift -*-===//
+//
+// This source file is part of the Swift Numerics open source project
+//
+// Copyright (c) 2020 Apple Inc. and the Swift Numerics project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+//
+//===----------------------------------------------------------------------===//
+
+extension Numeric where Magnitude: FloatingPoint {
+  /// Compares `self` and `other` for approximate equality with a specified
+  /// relative tolerance and implicit absolute tolerance.
+  ///
+  /// `true` if `self` and `other` are equal, or if they are finite and
+  /// ```
+  /// (self - other).magnitude ≤ relativeTolerance * scale
+  /// ```
+  /// where `scale` is
+  /// ```
+  /// max(self.magnitude, other.magnitude, .leastNormalMagnitude)
+  /// ```
+  ///
+  /// The default value of `relativeTolerance` is `.ulpOfOne.squareRoot()`,
+  /// which corresponds to expecting "about half the digits" in the computed results to be good.
+  /// This is the usual guidance in numerical analysis, if you don't know anything about the
+  /// computation being performed, but is not suitable for all use cases.
+  ///
+  /// Mathematical Properties:
+  /// -
```

Comment:
```suggestion
  ///
```

Reply:
Unintentional empty bullet?

Reply:
It makes Xcode quick help format it as a header:
<img width="253" alt="Screen Shot 2020-07-14 at 3 25 44 PM" src="https://user-images.githubusercontent.com/641446/87467693-5a846f00-c5e6-11ea-8bed-411f8abdc92f.png">
vs
<img width="195" alt="Screen Shot 2020-07-14 at 3 25 59 PM" src="https://user-images.githubusercontent.com/641446/87467711-5fe1b980-c5e6-11ea-9683-204eaff4f41f.png">

Reply:
Could I suggest maybe extending the underline to make that clear?